client/src/redux/sagas/intro.sagas.js

```diff
@@ -19,10 +45,21 @@ export function* watchFetchIntroMessages() {
   yield takeLeading('FETCH_INTRO', fetchIntroMessages);
 }
 
+// Write IntroMessages to the database and update the redux store.
 export function* saveIntroMessages(action) {
   yield put({ type: 'INTRO_SAVE_STARTED' });
   try {
-    localStorage.setItem('introMessages', JSON.stringify(action.payload));
+    const introMessages = {};
+    Object.entries(action.payload).forEach(([component, unread]) => {
+      introMessages[component] = { unread };
+    });
+
+    yield call(fetchUtil, {
+      url: '/api/intromessages/',
+      method: 'PUT',
+      body: JSON.stringify(introMessages)
```

Comment:
Could we simplify this by just sending over the JSON as it is? I don't expect we'll need more params for these than just `unread`, so we can assume that implicitly.
```suggestion
    yield call(fetchUtil, {
      url: '/api/intromessages/',
      method: 'PUT',
      body: JSON.stringify(action.payload)
```